ambry-store/src/test/java/com.github.ambry.store/IndexValueUtils.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.utils.Utils;
+import java.nio.ByteBuffer;
+
+
+/**
+ * IndexValue utils to assist in constructing {@link IndexValue} in different versions. For
+ * {@link PersistentIndex#VERSION_1}, all args are used to construct the {@link IndexValue} and for
+ * {@link PersistentIndex#VERSION_0} unnecessary args are dropped to construct the {@link IndexValue}
+ */
+class IndexValueUtils {
```

Comment:
I'm not convinced that a class is required for this. These functions can be static functions in `IndexValueTest`. I don't think we should introduce new classes to do this.